digitalocean/Database.py

```diff
@@ -0,0 +1,218 @@
+# -*- coding: utf-8 -*-
+from .baseapi import BaseAPI, POST, DELETE, PUT
+
+__SIZE_SLUG__ = [
+    'db-s-1vcpu-1gb',
+    'db-s-1vcpu-2gb',
+    'db-s-2vcpu-4gb',
+    'db-s-4vcpu-8gb',
+    'db-s-6vcpu-16gb',
+    'db-s-8vcpu-32gb',
+    'db-s-16vcpu-64gb'
+]
+
+__DB_ENGINES__ = {
+    'pq': {
```

Comment:
I think this should be `pg` 😉 

```suggestion
    'pg': {
```